tracer/src/Datadog.Trace/Telemetry/Collectors/ConfigurationTelemetryCollector.cs

```diff
@@ -113,6 +113,7 @@ public ICollection<TelemetryValue> GetConfigurationData()
                 new(ConfigTelemetryData.Platform, value: FrameworkDescription.Instance.ProcessArchitecture),
                 new(ConfigTelemetryData.Enabled, value: settings.TraceEnabled),
                 new(ConfigTelemetryData.AgentUrl, value: settings.Exporter.AgentUri.ToString()),
+                new(ConfigTelemetryData.AgentTraceTransport, value: settings.Exporter.TracesTransport.ToString()),
```

Comment:
Can you augment `TelemetryHelpers.cs` to test this configuration and other default configuration settings?

Reply:
That doesn't have to happen in this PR but I figured it would be good to make some more assertions on our default telemetry 😄